aws_lambda_builders/utils.py

```diff
@@ -74,8 +74,12 @@ def copytree(source, destination, ignore=None, include=None):
             LOG.debug("File (%s) doesn't satisfy the include rule, skipping it", name)
             continue
 
-        if os.path.isdir(new_source):
-            copytree(new_source, new_destination, ignore=ignore, include=include)
+        if os.path.islink(new_source) and maintain_symlinks:
+            linkto = os.readlink(new_source)
+            create_symlink_or_copy(linkto, new_destination)
+            shutil.copystat(new_source, new_destination, follow_symlinks=False)
```

Comment:
similar to previous question, why we create symlink and also copy ?

Reply:
answered here: https://github.com/aws/aws-lambda-builders/pull/461#discussion_r1122153103